docs/security.rst

```diff
@@ -310,3 +310,20 @@ standard port 443, you'll need to configure that too. Be aware that super user p
     # Optionally, set the server to listen on the standard SSL port.
     web_server_port = 443
     base_url = http://<hostname or IP>:443
+
+Impersonation
+'''''''''''''
+
+Airflow has the ability to impersonate a unix user while running task
+instances based on the task's ``run_as_user`` parameter, which takes a user's name.
+
+*NOTE* For impersonations to work, Airflow must be run with `sudo` as subtasks are run
```

Comment:
I did some brief reading on this and it seems like a bit of an uphill battle in terms of security, e.g.:
https://blog.compass-security.com/2012/10/dangerous-sudoers-entries-part-4-wildcards/

I will try to make the sudoers file as safe as possible, but given that there might be workarounds, it might be safer to just allow the airflow user to run as root (then at least the user will not make false assumptions regarding safety).